sw/airborne/firmwares/rotorcraft/stabilization/stabilization_rate_indi.h

```diff
@@ -26,12 +27,15 @@
 #ifndef STABILIZATION_RATE_INDI
 #define STABILIZATION_RATE_INDI
 
+/* access declarations of basic functions from "stabilization_rate.h" file */
+#include "firmwares/rotorcraft/stabilization/stabilization_rate.h"
+
 #include "math/pprz_algebra_int.h"
 
-extern void stabilization_rate_init(void);
-extern void stabilization_rate_read_rc(void);
-extern void stabilization_rate_read_rc_switched_sticks(void);
-extern void stabilization_rate_run(bool in_flight);
-extern void stabilization_rate_enter(void);
+extern void stabilization_indi_rate_set_setpoint_i(struct Int32Rates *pqr);
+extern void stabilization_rate_run(bool in_flight, struct Int32Rates rates_sp);
```

Comment:
This might conflict with the same function declared in `stabilization_rate.h`. So here it should be `stabilization_indi_rate_run`. And in the C code, you need to implement both function with one simply calling the second:
```
void stabilization_rate_run(bool in_flight)
{
  ...
  stabilization_indi_rate_run(in_flight, rate_ref);
}
```